tools/type_whisperer/file_descriptor_set_text_gen.py

```diff
@@ -0,0 +1,23 @@
+# Generate a text proto from a given list of FileDescriptorSets.
```

Comment:
I think you can embed them in binary by encoding them in proto_cc_source?

Reply:
I've added a TODO. I think I rather keep this simple for the current set of work, as plain text proto is easier to debug.